containers/search/SearchDropdown.js

```diff
@@ -24,6 +32,12 @@ const SearchDropdown = ({
         close();
     };
 
+    const handleReset = () => {
+        updateSearch('');
```

Comment:
not sure if we should guard against the default reset action here, like we do for `handleSubmit`, since `ResetButton` is of type "reset"

Reply:
I asked myself too. But after some tests, I seen any problem.

Reply:
You made me doubt. I added the prevent default just to be sure.